cmd/thanos/main.go

```diff
@@ -227,6 +228,19 @@ func defaultGRPCServerOpts(logger log.Logger, reg *prometheus.Registry, tracer o
 			grpc_recovery.StreamServerInterceptor(grpc_recovery.WithRecoveryHandler(grpcPanicRecoveryHandler)),
 		),
 	}
+
+	if insecure {
+		return opts, nil
+	}
+
+	tcreds, err := credentials.NewServerTLSFromFile(cert, key)
```

Comment:
`tlsCreds` instead?

Reply:
the t there is for transport, but tls would not be totally inaccurate.

Reply:
so then `transport` is fine as well (: just `t` could be ambigous

Reply:
opted fore `creds`, since we aren't using perRPC creds, that should be ok.